pymc/model_graph.py

```diff
@@ -102,11 +102,42 @@ def get_parents(self, var: TensorVariable) -> Set[VarName]:
         parents = self._get_ancestors(var, func)
         return self._filter_parents(var, parents)
 
-    def make_compute_graph(self) -> Dict[str, Set[VarName]]:
+    def vars_to_plot(self, selected_vars=None):
```

Comment:
```suggestion
    def vars_to_plot(self, var_names: Optional[Iterable[str]]=None) -> List[str]:
```
Reading the code I was confused because `selected_vars` are the _names_, right?